docker-image-src/5/neo4jlabs-plugins.json

```diff
@@ -1,7 +1,12 @@
 {
   "apoc": {
     "location": "/var/lib/neo4j/labs/apoc-*-core.jar",
-    "versions": "https://github.com/neo4j/apoc/blob/master/versions.json",
+    "properties": {
+      "dbms.security.procedures.unrestricted": "apoc.*"
+    }
+  },
+  "apoc-core": {
+    "location": "/var/lib/neo4j/labs/apoc-*-core.jar",
     "properties": {
       "dbms.security.procedures.unrestricted": "apoc.*"
     }
```

Comment:
I'm hesitant to introduce this option, because I believe some team members might be against it and that is why the option was removed between 4.x and 5.x 

To me, the option really just acts as an alias for apoc, and I would like to suggest adding it because I think it would be nice to be have a backwards compatible between Neo4j 4.x and 5.x.